package.json

```diff
@@ -423,14 +422,26 @@
     "@types/slate": "0.47.11",
     "@rushstack/rig-package": "0.5.1",
     "@rushstack/ts-command-line": "4.16.1",
-    "ngtemplate-loader/loader-utils": "^2.0.0",
```

Comment:
While pnpm does support both the resolutions and pnpm.overrides fields, pnpm chokes on this syntax for overriding transitive dependencies, so I've removed this for now.

I don't think we actually build angular templates any more, so it probably doesn't hurt to just remove this.